.github/workflows/ci-docker-build.yml

```diff
@@ -44,7 +44,16 @@ jobs:
 
     - uses: docker/setup-qemu-action@e81a89b1732b9c48d79cd809d8d81d79c4647a18
 
+    - name: Build and upload only linux/amd64 docker image for Pull Request
+      if: github.ref_name != 'main'
+      run: bash scripts/build-upload-docker-images.sh
+      env:
+        PLATFORMS: "linux/amd64"
+        DOCKERHUB_TOKEN: ${{ secrets.DOCKERHUB_TOKEN }}
+        QUAY_TOKEN: ${{ secrets.QUAY_TOKEN }}
```

Comment:
Since we do not upload images for PRs, I think it's not necessary to pass tokens

Reply:
Sounds good👍